book/src/keymap.md

```diff
@@ -452,6 +452,13 @@ Keys to use within picker. Remapping currently not supported.
 | `Ctrl-t`                     | Toggle preview                                             |
 | `Escape`, `Ctrl-c`           | Close picker                                               |
 
+In pickers, you can filter specific columns by prefixing the column name with `%`.
+All words after a `%<column>` term are treated as filters for that column.
+The base filter is a regex, while column filters are a space-separated of literals.
```

Comment:
Actually columns use fuzzy filtering. There are two exceptions: the default columns for global_search and the LSP workspace_symbol_picker. global_search's default column is a regex and workspace_symbol_picker's is sent to the language server so it can do its own filtering.